Dockerfile.template

```diff
@@ -34,7 +34,7 @@ RUN echo 'deb http://www.apache.org/dist/cassandra/debian %%CASSANDRA_DIST%%x ma
 ENV CASSANDRA_VERSION %%CASSANDRA_VERSION%%
 
 RUN apt-get update \
-	&& apt-get install -y cassandra="$CASSANDRA_VERSION" \
+	&& apt-get install -y cassandra="$CASSANDRA_VERSION" libjemalloc1 \
```

Comment:
Can we instead add a line above `ENV GPG_KEYS` so that all the versions can share the cached layer?

ie, something like this:
```dockerfile
...
 	&& apt-get purge -y --auto-remove ca-certificates wget

# "jemalloc shared library could not be preloaded to speed up memory allocations"
RUN apt-get update && apt-get install -y --no-install-recommends libjemalloc1 && rm -rf /var/lib/apt/lists/*

 # https://wiki.apache.org/cassandra/DebianPackaging#Adding_Repository_Keys
 ENV GPG_KEYS \
...
```